test/test_ops.py

```diff
@@ -1399,14 +1404,25 @@ def test_video_encoder_round_trip(self, tmp_path, format):
         ):
             pytest.skip("Codec for webm is not available in this FFmpeg installation.")
         asset = TEST_SRC_2_720P
-        source_frames = self.decode(str(asset.path)).data
+        source_frames = self.decode(file_path=str(asset.path)).data
 
-        encoded_path = str(tmp_path / f"encoder_output.{format}")
         frame_rate = 30  # Frame rate is fixed with num frames decoded
-        encode_video_to_file(
-            frames=source_frames, frame_rate=frame_rate, filename=encoded_path, crf=0
-        )
-        round_trip_frames = self.decode(encoded_path).data
+        if output_method == "to_file":
+            encoded_path = str(tmp_path / f"encoder_output.{format}")
+            encode_video_to_file(
+                frames=source_frames,
+                frame_rate=frame_rate,
+                filename=encoded_path,
+                crf=0,
+            )
+            round_trip_frames = self.decode(file_path=encoded_path).data
+        else:  # to_tensor
+            format = "matroska" if format == "mkv" else format
+            encoded_tensor = encode_video_to_tensor(
+                source_frames, frame_rate, format, crf=0
+            )
+            round_trip_frames = self.decode(tensor=encoded_tensor).data
+
```

Comment:
Instead of passing frame_rate and crf to both methods, let's have those inside a `params` dict as done in the AudioEncoder tests. It will make it clearer what are common parameters (that are only set once) vs the parameters that are specific to each method.

https://github.com/meta-pytorch/torchcodec/blob/b084768e63d8d200b2945cf659fc930cfe72b04b/test/test_encoders.py#L278-L291